src/message_filter.rs

```diff
@@ -15,6 +15,7 @@
 // Please review the Licences for the specific language governing permissions and limitations
 // relating to use of the SAFE Network Software.
 
+#![allow(deprecated)]
```

Comment:
I've actually tried to mark only the line importing SipHasher, but apparently it doesn't work that way. I didn't have any better idea for how to do this - should I add a todo, then?